lib/components/src/blocks/ArgsTable/ArgsTable.tsx

```diff
@@ -398,7 +399,7 @@ export const ArgsTable: FC<ArgsTableProps> = (props) => {
   const { rows, args } = 'rows' in props ? props : argsTableLoadingData;
 
   const groups = groupRows(
-    pickBy(rows, (row) => !row?.table?.disable),
+    pickBy(rows, (row) => !row?.table?.disable && includeConditionalArg(row, args)),
```

Comment:
Do we still need the code `!row?.table?.disable` where we are changing the arg that we pass to controls from:
```
foo:{
  table:{
    disable: true,
  }
}
```
to
```
foo:{
  removeIf: true,
}
```

Reply:
@ethriel3695 yes, otherwise it's a breaking change. (If i understand your question). We try to only make breaking changes in major releases, and give some deprecation warning in advance